lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java

```diff
@@ -139,9 +141,11 @@ private AAALighty initAAA(RncAAAConfiguration config, LightyServices services) {
             this.jettyServerBuilder);
     }
 
-    private SwaggerLighty initSwaggerLighty(RncRestConfConfiguration config, LightyServerBuilder jettyServerBuilder, LightyServices services) {
+    private SwaggerLighty initSwaggerLighty(RncRestConfConfiguration config,
+                                            LightyServerBuilder jettySrvBuilder,
+                                            LightyServices services) {
         final RncRestConfConfiguration confConf = RncRestConfConfigUtils.getRestConfConfiguration(config, services);
-        return new SwaggerLighty(confConf, jettyServerBuilder, services);
+        return new SwaggerLighty(confConf, jettySrvBuilder, services);
```

Comment:
No need to rename jettyServerBuilder.

Reply:
Your compilation / build rules need it. Since there is a jettyServerBuilder member in this class, it deems it as hiding a member. So if I roll-back to _jettyServerBuilder_ your CI will fail. What is your opinion?

Reply:
OK, ACK.